PF/Assignment1/assignment3.java

```diff
@@ -0,0 +1,60 @@
+package programing_fundamental;
+
+import java.util.*;
+
+public class assignment3 {
+	public int[] longestSequence(int input[])
+	{
+		int n=input.length;
+		int buff1[]=new int[n];
+		int buff2[]=new int[n];
+		int count1=1,count2=1;
+		int k=0;
+		int i,j,l;
+		buff1[k]=input[0];
+		for(i=0;i<n;)
```

Comment:
ACTION: Instead of using two loops and two array and these many variables, Just iterate input array once and maintain position and size of longest sequence. and then after the loop create a new array by using the position and size of longest sequence.